pretender.js

```diff
@@ -261,9 +261,15 @@ function interceptor(pretender, nativeRequest) {
   FakeRequest.prototype = proto;
 
   if (nativeRequest.prototype._passthroughCheck) {
-    throw new Error('You created a second Pretender instance while there was already one running. ' +
-          'Running two Pretender servers at once will lead to unexpected results!' +
-          'Please call .shutdown() on your instances when you no longer need them to respond.');
+    var debugMessage  = '[Pretender.js] You created a second Pretender instance while there was already one running. ';
+    debugMessage     += 'Running two Pretender servers at once will lead to unexpected results!';
+    debugMessage     += 'Please call .shutdown() on your instances when you no longer need them to respond.';
```

Comment:
i think the style of the original change is better for concatenating a long message, e.g.:

```js
console.warn('DEPRECATED: You created ' +
  'a second Pretender instance ' +
  'while a second one ...');
```